test/suite/test_timestamp22.py

```diff
@@ -147,10 +148,10 @@ def timestamp_txn_config(commit_ts, running_commit_ts):
                     # It's possible this will succeed, we'll check below.
                     this_commit_ts = self.gen_ts(commit_ts)
 
-                    # OOD does not work with prepared updates. Hence, the commit ts should always be
-                    # greater than the last durable ts.
-                    if this_commit_ts <= self.last_durable:
-                        this_commit_ts = self.last_durable + 1
+                    # OOO does not work. Hence, the commit ts should always be
+                    # greater than the last commit ts.
```

Comment:
Maybe "is not allowed" instead of "does not work"?

Reply:
Fixed.